source/adressering-og-varsler.rst

```diff
@@ -0,0 +1,253 @@
+Signeringsfrist 1. varsel: e-post 2. varsel: e-post 3. varsel: SMS
+=============== ================= ================= =================
+0-24 timer      Ved aktivering    -                 Ved aktivering
+2-3 dager       Ved aktivering    1 dag før frist   1 dag før frist
+4-5 dager       Ved aktivering    2 dager før frist 1 dag før frist
+6-9 dager       Ved aktivering    3 dager før frist 2 dager før frist
+10+ dager       Ved aktivering    5 dager før frist 2 dager før frist
+=============== ================= ================= =================
+
+.. raw:: html
+
+   <!-- Tabellen er generert vha. http://www.tablesgenerator.com/markdown_tables -->
+
+Hvis undertegner adresseres på kun e-postadresse eller kun mobilbummer, eller det kun finnes en av delene, sendes det ut inntil 2 varsler:
+
+=============== ===================== =====================
+Signeringsfrist 1. varsel: e-post/SMS 2. varsel: e-post/SMS
+=============== ===================== =====================
+0-24 timer      Ved aktivering        -
+2-3 dager       Ved aktivering        1 dag før frist
+4-5 dager       Ved aktivering        2 dager før frist
+6-9 dager       Ved aktivering        3 dager før frist
+10+ dager       Ved aktivering        5 dager før frist
+=============== ===================== =====================
+
+.. raw:: html
+
+   <!-- Tabellen er generert vha. http://www.tablesgenerator.com/markdown_tables -->
+
+.. NOTE:: SMS sendes ikke ut mellom klokken 22:00 og 08:00, med mindre oppdraget opprettes på natten og fristen er så kort at det er nødvendig med umiddelbar utsending.
+
+.. NOTE:: Hvis avsender *utvider signeringsfristen* "kanselleres" alle planlagte varsler for oppdraget. Det blir da generert nye varsler som da blir sendt ut på tidspunkt knyttet til den nye fristen.
+
+
+Varseltekster for undertegner
+=============================
+
+Innholdet i varslene som blir sendt ut er predefinert og ikke mulig å endre på for deg som avsender. Varseltekstene er hovedsakelig forskjellig ut fra om adresseringen til undertegner er med eller uten fødselsnummer. I tillegg er det et par ulikheter som kommer fra variablene
+- hvilken kanal de sendes i (e-post/SMS)
+- sektor som avsender sender fra (privat eller offentlig)
+- antall undertegnere på oppdraget
+
+
+Varsel om dokument til signering, ved adressering med fødselsnummer
+____________________________________________________________________
+
+..  tabs::
+
+    ..  tab:: Epost-tekst 1. varsel
+
+        **Epost-emne**: Dokument til signering fra *Avsender*
+
+        Hei!
+
+        Du har fått en forespørsel om å signere et dokument fra *Avsender*: *Tittel på dokumentet*. Dokumentet må signeres innen *Signeringsfrist*. Du kan signere med *disse elektroniske e-IDene*.
+
+        Logg deg inn på *signering.posten.no/logginn* for å signere dokumentet.
+
+        Hilsen Posten
+    
+      
+    ..  tab:: Epost-tekst 2. varsel
+
+        **Epost-emne**: Påminnelse: Dokument til signering fra *Avsender*
```

Comment:
Det leser litt stygt, men det bør være E-post-emne. Kan vi kanskje bare bruke **Emne:**?

Reply:
Emne it is!